lib/tasks/web.rake

```diff
@@ -74,6 +74,33 @@ namespace :slack do
           'bin/commands.rb',
           commands_template.result(files: data.keys.map { |key| key.tr('.', '_') })
         )
+
+        ### Errors
+
+        # Use the official spec to pull a list of errors and create classes for each
+
+        spec = JSON.parse(open('https://raw.githubusercontent.com/slackapi/slack-api-specs/master/web-api/slack_web_openapi_v2.json').read)
```

Comment:
Do you think we can do https://github.com/slack-ruby/slack-ruby-client/issues/204 as well soon? If so I wonder whether we should be downloading and committing this JSON to have stable history against which version of the API the generator ran?

Reply:
I definitely thought about #204 but decided it was too much for me to bite off right now. But I can definitely put something in to write and commit the JSON. Are you thinking just a `.json` file somewhere in the repo that has the exact contents of the spec?

Reply:
We bring slack-api-ref into lib/slack/web/api/slack-api-ref, maybe just lib/slack-api-specs?